app/controllers/UserController.scala

```diff
@@ -0,0 +1,35 @@
+package controllers
+
+import scala.concurrent.Future
+
+import com.mohiva.play.silhouette.api.actions.SecuredRequest
+import com.mohiva.play.silhouette.api.Silhouette
+import javax.inject._
+import play.api.libs.json._
+import play.api.mvc._
+
+import models.User
+import utils.auth.DefaultEnv
+
+class UserController @Inject() (
+  cc: ControllerComponents,
+  silhouette: Silhouette[DefaultEnv]
+) extends AbstractController(cc) {
+
+  implicit val userWrite = new Writes[User] {
```

Comment:
Idiomatically, where would serializers like this live?

Reply:
er not super sure. here are two options.

https://stackoverflow.com/questions/36082960/playframework-where-to-put-json-writes-and-reads-for-reuse

I've also seen like this

https://github.com/sivaprakashniet/dyce/blob/b80186ba8ffc23154738f4cace4a8c3b98cb5ec1/app/controllers/ControllerImplicits.scala

Reply:
Oh I kinda like putting it in the model companion object.